data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/teleport/teleport_no_tunnel.cfg

```diff
@@ -0,0 +1,61 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [teleport]
+##
+# Actions:
+# Give side 1 units the ability to teleport but that has no tunnel defined
+# Move side 1's leader to the edge of the map
+# Place a village next to side 1's leader and side 2's leader
```

Comment:
This test can also pass because capturing 1,2 costs the unit all of its movement.